_posts/2019-03-22-team-time.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: "Team Time: developing efficient team communication"
+author: richard_harper
+---
+
+![Team Time!]({{ "/images/direction.png" }})
+
+# Avoiding the overrunning daily Stand-Up
+
+At Infinity Works we use an agile way of working which usually includes a daily Stand-Up. This 5-10 minute timeboxed meeting allows the team to provide an update and more importantly raise any issues they are facing. Our team uses the format of what they did yesterday (to achieve the team’s goal), what they are working on that day (to achieve the team’s goal), and anything that might prevent them from progressing with their work.
+
+Like most teams, it wasn’t uncommon for us to get side tracked or to start solutioning during a Stand-Up leading to long Stand-Up times and using up team member’s time who might have better things to be doing.
```

Comment:
I don't think "solutioning" is grammatically correct (https://english.stackexchange.com/questions/67242/is-solutioning-a-correct-word/67250). You could maybe say "solving the problem"/"discussing possible solutions"?

Reply:
Should be resolved please re-review